conf/mesos-site.xml

```diff
@@ -18,107 +18,37 @@
 
```

Comment:
Please don't modify the original conf/mesos-site.xml file.This will create a bug for users that are not using Mesos DNS. We should leave the original files intact and only add examples for users who want to use Mesos DNS.